devdoc/docs/ContributeDocs.md

```diff
@@ -0,0 +1,394 @@
+# Contributing Documentation to the HPCC Systems Platform Project
+
+This document is intended for anyone that wants to contribute
+documentation to our project. The first audience is platform developers,
+so we can streamline the process of documenting new features. However,
+these guidelines apply to anyone who wants to contribute to any of our
+documentation (Language Reference, Programmer’s Guide, etc.).
+
+This set of guidelines should help you understand the information needed
+to create sufficient documentation for any new feature you add. The good
+news is that we are all here to help you and support your writing
+efforts. We will help by advising you along the way, and by reviewing
+and editing your submissions.
+
+## Documenting a New Software Feature--Required and Optional Components
+
+When you create a new feature or function, clear documentation is
+crucial for both internal teams and external users. You worked hard on
+the feature, so it deserves proper notice and usage.
+
+We are an open-source project, so contributions are always welcome,
```

Comment:
The guidelines described in this file will apply to be community and non-community.  This will be published into our public github report, but I suggest removing "We are an open-source project.   We wouldn't want internal people to think this only applies to external docs.

Maybe something like ... "Contributions to the platform are always welcome, and we strongly encourage developers and users to contribute documenation"